samples/correlation-csharp/FunctionAppCorrelation/local.settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "APPINSIGHTS_INSTRUMENTATIONKEY": "f152bd88-9b36-43ba-a8a6-ebf921f9fc8f"
+  }
+}
```

Comment:
nit, empty line before EOF, there might be other place that need to updated, too.

Reply:
Thank you! I'll fix this on the samples branch!